View/AggregatedStageDamage/CalculatedStageDamageControl.xaml

```diff
@@ -4,45 +4,103 @@
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
              xmlns:utils="clr-namespace:HEC.FDA.View.Utilities"
+             xmlns:local="clr-namespace:HEC.FDA.View.AggregatedStageDamage"
              mc:Ignorable="d" 
              xmlns:twp="clr-namespace:HEC.FDA.View.TableWithPlot"
              d:DesignHeight="450" d:DesignWidth="800">
     <UserControl.Resources>
         <utils:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter" />
+        <local:StageDischargeVisibilityConverter x:Key="StageDischargeVisibilityConverter" />
```

Comment:
The StagedischargeVisibilityConverter isn't actually wired to Stage and discharge, you're just checking Bools. Could it be made more generic, or potentially just replaced with a bool to vis?

Reply:
This could be refactored to bind the visibility to a bool in the VM but then i think you would have to have all the same logic that is in my converter in the VM class. Somewhere you have to drill into the frequency object and check that bool. I chose to keep it all in the View rather than in the VM. I don't see a way to easily make this cleaner. To do it in the vm, I think you would have to put a method call in the setter of the selected frequency property to update the bool visibility property. I try to avoid calling things from the setters if i can. I view both approaches as equal and since this one is already written, i vote to keep it the way that it is.